torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -7,6 +7,7 @@
 #include <torch/csrc/jit/passes/common_subexpression_elimination.h>
 #include <torch/csrc/jit/passes/constant_pooling.h>
 #include <torch/csrc/jit/passes/dead_code_elimination.h>
+#include <torch/csrc/jit/passes/tensorexpr_fuser.h>
```

Comment:
It feels weird to have graph_fuser depend on tensorexpr_fuser.  Can we put these in like a fuser_utils or something?